src/hooks/useNotificationTypes.tsx

```diff
@@ -351,6 +357,42 @@ export const notificationTypes: NotificationTypeConfig[] = [
       return () => {};
     },
   },
+  {
+    type: NotificationType.ComplianceAlert,
+    useTrigger: ({ trigger }) => {
+      const stringGetter = useStringGetter();
+      const { complianceMessage, complianceState, complianceStatus } = useComplianceState();
+
+      useEffect(() => {
+        if (complianceState !== ComplianceStates.FULLACCESS) {
+          const displayData: NotificationDisplayData = {
+            icon: <$WarningIcon iconName={IconName.Warning} />,
+            title: stringGetter({ key: STRING_KEYS.COMPLIANCE_WARNING }),
+            body: complianceMessage,
+            toastSensitivity: 'foreground',
+            groupKey: NotificationType.ComplianceAlert,
+            withClose: false,
+          };
+
+          trigger(NotificationType.ComplianceAlert, displayData, []);
```

Comment:
do we need a more unique `id`?
currently when testing, if i had dismissed one alert for a certain status, i would not see the second toast for another -- should we handle that case?

Reply:
originally I figure since the notification is not dis-missable and I thought there would only be one at a time. but thinking on it more there can be more then one at a time. I'll add the status type to the Id